File: Collector/LegacyJava/Collector/Dimension/Inventory.java
package Collector.Dimension;

import java.util.LinkedList;

public class Inventory {
    private LinkedList<ItemStack> inventory = new LinkedList<ItemStack>();

    public LinkedList<ItemStack> getInventory() {
        return inventory;
    }

    public void setInventory(LinkedList<ItemStack> inventory) {
        this.inventory = inventory;
    }

    public void addItem(Block block){
        inventory.add((ItemStack) block);
    }

    public void removeItem(ItemStack itemStack){
        inventory.remove(itemStack);
    }
}

File: Collector/src/Dimension/Inventory.cs
using System.Collections.Generic;
using Collector.Dimension;
    
public class Inventory {
    private LinkedList<ItemStack> inventory = new LinkedList<ItemStack>();

    public LinkedList<ItemStack> GetInventory() {
        return inventory;
    }

    public void SetInventory(LinkedList<ItemStack> inventory) {
        this.inventory = inventory;
    }

    public void AddItem(Block block){
        inventory.AddFirst((ItemStack) block);
    }

    public void RemoveItem(ItemStack itemStack){
        inventory.Remove(itemStack);
    }
}